src/logger/index.ts

```diff
@@ -5,6 +5,8 @@ import * as Sentry from '@sentry/react-native';
 import * as env from '@/env';
 import { DebugContext } from '@/logger/debugContext';
 import { device } from '@/storage';
+import { push } from '@/logger/logDump';
+import { getExperimetalFlag, LOG_PUSH } from '@/config';
```

Comment:
@rainbow-me/app-team good example of why we shouldn't re-export files from an `index` file like this `... from '@/config'` — this is importing other crap from `@/config` index file that's then inadvertently breaking Jest.

I'll update this to import just the experimental config.